BibBuddy/app/src/main/java/de/bibbuddy/NoteDao.java

```diff
@@ -254,7 +254,7 @@ public String findTextById(Long id) {
             DatabaseHelper.TYPE, DatabaseHelper.TEXT,
             DatabaseHelper.CREATE_DATE, DatabaseHelper.MOD_DATE,
             DatabaseHelper.NOTE_FILE_ID},
-        DatabaseHelper._ID + "=?", new String[] {String.valueOf(id)},
+        DatabaseHelper._ID + " = ?", new String[] {String.valueOf(id)},
```

Comment:
In line 369 is used `"= ?"` without empty strings. Can you please change it like here, so that all are the same?